api/instance/v1/instance_utils.go

```diff
@@ -0,0 +1,94 @@
+package instance
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/scaleway/scaleway-sdk-go/utils"
+)
+
+// AttachIPRequest contains the parameters to attach an IP to a server
+type AttachIPRequest struct {
+	Zone     utils.Zone `json:"-"`
+	IPID     string     `json:"-"`
+	ServerID string     `json:"server"`
+}
+
+// AttachIP attaches an IP to a server.
+func (s *Api) AttachIP(req *AttachIPRequest, opts ...scw.RequestOption) (*Ip, error) {
```

Comment:
We could generate this function from the protobuf (only detachIP has a special behavior) 🤔

Reply:
How do you mean, generate? Yeah mean call `UpdateIP` from `AttachIP` or really generate?